File: VODDemos/WebPlayers/WebPlayers.Services/Asset.cs
namespace WebPlayers.Services
{
    using System;

    public class Asset
    {
        public string Id { get; set; }

        public string Name { get; set; }

        public string State { get; set; }

        public Uri HlsUri { get; set; }

        public Uri Hlsv3Uri { get; set; }

        public Uri MpegDashUri { get; set; }

        public Uri SmoothStreamingUri { get; set; }

        public Uri HighQualityMp4Uri { get; set; }

        public Uri MidQualityMp4Uri { get; set; }

        public Uri LowQualityMp4Uri { get; set; }

        public Uri ThumbnailUri { get; set; }
    }
}

File: VODDemos/WebPlayers/WebPlayers/Scripts/app/Models/Asset.ts
module Models {
    export class Asset {
        constructor(
            public id: string,
            public name: string,
            public status: string,
            public mpegDashUri: string,
            public hlsUri: string,
            public hlsv3Uri: string,
            public smoothStreamingUri: string,
            public thumbnailUri: string,
            public highQualityMp4Uri: string,
            public midQualityMp4Uri: string,
            public lowQualityMp4Uri: string) {
        }
    }
}